### ISSUE TITLE
Warning at writing to decrease modes

### ISSUE THREAD

**Issue by Pittini** [rg-engineering/ioBroker.heatingcontrol#86]
if i try to write via script to:

> heatingcontrol.0.Profiles.0.Room.AbsentDecrease
or any other decrease values

i got the warning:
`You are assigning a number to the state "heatingcontrol.0.Profiles.0.Bad.AbsentDecrease" which expects a float. Please fix your code to use a float or change the state type to number. `

The field is defined as float. Why? It seems to be impossible to write via script at float.  Even if i try a simple

```
let test = 5
setState("heatingcontrol.0.Profiles.0.Bad.AbsentDecrease", test)
```

or

```
let test = toFloat(5)
setState("heatingcontrol.0.Profiles.0.Bad.AbsentDecrease", test)
```

or

```
let test = parseFloat("5")
setState("heatingcontrol.0.Profiles.0.Bad.AbsentDecrease", test)
```

i ever get the warning. Isnt it possible to define those fields as number?

**Comment by rg-engineering** [comment]
I change all to "number"

**Comment by rg-engineering** [comment]
solved with 0.3.19
